src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -159,6 +159,10 @@ public string DataLakeAudience
             HelpMessage = "The endpoint to use when communicating with the Azure Log Analytics API.")]
         public string AzureOperationalInsightsEndpoint { get; set; }
 
+        [Parameter(Position = 23, Mandatory = false, ValueFromPipelineByPropertyName = true, 
```

Comment:
Non mandatory param, can you remove the position attribute on it?

Reply:
fixed